From a27c9d3621facb03a450422910fe2d5835d7d0d6 Mon Sep 17 00:00:00 2001
From: Theodore Olsauskas-Warren <sauski@google.com>
Date: Tue, 1 Apr 2025 08:39:49 -0700
Subject: [PATCH] patched

TODO(crbug.com/408276016): Upstream this version bump into the ff
repository.

---
 .../rust/chromium_crates_io/vendor/ff-0.13.1/Cargo.toml       | 4 ++--
 .../chromium_crates_io/vendor/ff-0.13.1/rust-toolchain.toml   | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/third_party/rust/chromium_crates_io/vendor/ff-0.13.1/Cargo.toml b/third_party/rust/chromium_crates_io/vendor/ff-0.13.1/Cargo.toml
index 3b025f826d27d..9853b82cbb651 100644
--- a/third_party/rust/chromium_crates_io/vendor/ff-0.13.1/Cargo.toml
+++ b/third_party/rust/chromium_crates_io/vendor/ff-0.13.1/Cargo.toml
@@ -11,7 +11,7 @@
 
 [package]
 edition = "2021"
-rust-version = "1.56"
+rust-version = "1.63"
 name = "ff"
 version = "0.13.1"
 authors = ["Sean Bowe <ewillbefull@gmail.com>", "Jack Grigg <thestr4d@gmail.com>"]
@@ -44,7 +44,7 @@ version = "0.13.1"
 optional = true
 
 [dependencies.rand_core]
-version = "0.6"
+version = "0.9"
 default-features = false
 
 [dependencies.subtle]
diff --git a/third_party/rust/chromium_crates_io/vendor/ff-0.13.1/rust-toolchain.toml b/third_party/rust/chromium_crates_io/vendor/ff-0.13.1/rust-toolchain.toml
index de43b23004fa5..3eebdfe1d3e95 100644
--- a/third_party/rust/chromium_crates_io/vendor/ff-0.13.1/rust-toolchain.toml
+++ b/third_party/rust/chromium_crates_io/vendor/ff-0.13.1/rust-toolchain.toml
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.56.0"
+channel = "1.63.0"
 components = [ "clippy", "rustfmt" ]
-- 
2.49.0.472.ge94155a9ec-goog

